lib/phoenix_swagger/plug/validate_plug.ex

```diff
@@ -97,11 +97,24 @@ defmodule PhoenixSwagger.Plug.Validate do
       for parameter <- schema.schema["parameters"],
           parameter["type"] != nil,
           parameter["in"] in ["query", "path"] do
-        {parameter["type"], parameter["name"], conn.params[parameter["name"]], parameter["required"]}
+        {parameter["type"], parameter["name"], get_nested_name(conn.params, parameter["name"]), parameter["required"]}
       end
     validate_query_params(parameters)
   end
 
+  defp get_nested_map(params = nil, _), do: params
```

Comment:
Can we rename this function `get_in_nested` ? As a hint that it acts like `Kernel.get_in` but has a nested key.